### PR TITLE
Feature/nuc 133 transfer flexbox layout

### PR DESCRIPTION
**Pull request by ercang**
image
image
image
image
image

### CODE REVIEW

File: docs/.vuepress/components/FlexboxLayoutLiveDemo.vue [hunk 1]
```diff
@@ -0,0 +1,300 @@
+<template>
+  <StackLayout>
+    <Label text="Container Properties" class="option-title"/>
+
+    <StackLayout orientation="horizontal">
+      <Label text="Width" class="option-label"/>
+      <input id="flexboxlayout_containerwidth_input" type="text" v-model="containerWidth">
```
Comment (ahmetceylan) [inline docs/.vuepress/components/FlexboxLayoutLiveDemo.vue]: please replace all inputs with textfield component

File: docs/.vuepress/components/FlexboxLayoutLiveDemo.vue [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+            <option>blue</option>
+            <option>yellow</option>
+            <option>cyan</option>
+            <option>orange</option>
+            <option>gray</option>
+          </select>
+        </StackLayout>
+        
+        <StackLayout orientation="horizontal" class="editor-param">
+          <button :id="'docklayout_childreneditor_removechild' + index" @click="removeChildAtIndex(index)">❌</button>
```
Comment (ahmetceylan) [inline docs/.vuepress/components/FlexboxLayoutLiveDemo.vue]: please replace buttons with vuexp button

File: docs/layouts/flexbox-layout.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+<FlexboxLayoutColumnReverseDoc />
+</DocExampleBox>
+
+## Props
+
+| Name           | Type   | Default      | Web | Mobile |
+| -------------- | ------ | ------------ | --- | ------ |
+| flexDirection  | String | `row`        | ✔   | ✔      |
```
Comment (ahmetceylan) [inline docs/layouts/flexbox-layout.md]: please add description of props and events

File: docs/.vuepress/components/FlexboxLayoutLiveDemo.vue (hunk 1 above)
Comment (ercang) [reply docs/.vuepress/components/FlexboxLayoutLiveDemo.vue]: fixed

File: docs/layouts/flexbox-layout.md (hunk 3 above)
Comment (ercang) [reply docs/layouts/flexbox-layout.md]: updated docs

File: docs/.vuepress/components/FlexboxLayoutLiveDemo.vue (hunk 2 above)
Comment (ercang) [reply docs/.vuepress/components/FlexboxLayoutLiveDemo.vue]: fixed

**Pull request merged**